app/lib/annual_report/percentiles.rb

```diff
@@ -1,62 +1,37 @@
 # frozen_string_literal: true
 
 class AnnualReport::Percentiles < AnnualReport::Source
+  def self.prepare(year)
+    AnnualReport::StatusesPerAccountCount.connection.exec_query(<<~SQL.squish, nil, [year, Mastodon::Snowflake.id_at(DateTime.new(year).beginning_of_year), Mastodon::Snowflake.id_at(DateTime.new(year).end_of_year)])
```

Comment:
I think we have base class convenience methods for the beginning/ending values.

Separately, this whole thing could use the framework upsert class method? (I think)